docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -94,20 +94,38 @@ genesis file.
 
 ### Extra data
 
-The `extraData` property is RLP encoded. RLP encoding is a space efficient object serialization
-scheme used in Ethereum. To generate the `extraData` RLP string for inclusion in the genesis file,
-use the [`rlp encode`](../../../Reference/CLI/CLI-Subcommands.md#rlp) Besu subcommand.
+The `extraData` property is an RLP encoding of:
+
+* 32 bytes of vanity data.
+* A list of initial validator addresses (at least one initial validator is required).
```

Comment:
This not just the initial validator addresses. It also represents the validators for a particular height. But for the genesis block it is the initial validators.